flash/vision/backbones.py

```diff
@@ -140,3 +147,21 @@ def torchvision_backbone_and_num_features(model_name: str, pretrained: bool = Tr
         return backbone, num_features
 
     raise ValueError(f"{model_name} is not supported yet.")
+
+
+def timm_backbone_and_num_features(model_name: str, pretrained: bool = True) -> Tuple[nn.Module, int]:
+    """
+    >>> timm_backbone_and_num_features('resnet18')  # doctest: +ELLIPSIS
+    (ResNet(...), 512)
+    >>> timm_backbone_and_num_features('mobilenetv3_large_100')  # doctest: +ELLIPSIS
+    (MobileNetV3(...), 1280)
+    """
+
+    if model_name in timm.list_models():
+        backbone = timm.create_model(model_name, pretrained=pretrained, num_classes=0,
```

Comment:
Why num_classes = 0 ?

Reply:
this method is suggested to create a model without a classification head in the timm docs. https://rwightman.github.io/pytorch-image-models/feature_extraction/#create-with-no-classifier-and-pooling